aries-site/src/pages/components/checkboxgroup.js

```diff
@@ -0,0 +1,163 @@
+          figma="https://www.figma.com/file/zkQOQRUt8frxiSo67kTkfE/HPE-Checkbox-Group-Component?node-id=174%3A23"
+          designer="https://designer.grommet.io/checkboxgroup?id=HPE-design-system-hpedesignsystem-hpe-com"
+        >
+          <CheckBoxGroupSimpleExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          CheckBoxGroup should be used to present a group of options to the
+          user. The user can select a single or multiple options.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Best Practices" level={3} gap="small">
+        <BulletedList
+          level={3}
+          items={[
+            `CheckBoxGroup should be used to select zero or more of options 
+              from a list.`,
+            'In a CheckBoxGroup, options are independent from one another.',
+            `Settings changed by checkboxes should not take effect until the 
+              user clicks to proceed.`,
+            `It is recommended to have only 10 or fewer options in a 
+              CheckBoxGroup. Use multiple CheckboxGroups if necessary.`,
+          ]}
+        />
+      </Subsection>
+      <Subsection name="Usage" level={3} gap="small">
+        <Text weight="bold">The dos for CheckBoxGroup:</Text>
+        <BulletedList
+          level={3}
+          items={[
+            `Always label CheckBoxGroups with short and concise labels.`,
+            `Give every CheckBox in the group a short, but descriptive 
+              value.`,
+            `Clearly separate Checkbox Groups from other groups on the same
+              page.`,
+            `Organize Checkboxes in a logical order by grouping related
+              options.`,
+            `Consider placing most common options first in the group.`,
+          ]}
+        />
+        <Text weight="bold">The don'ts for CheckBoxGroup:</Text>
+        <BulletedList
+          level={3}
+          items={[
+            `Don’t align checkboxes horizontally in a group.`,
+            `Refrain from using CheckboxGroup when only one item can be 
+              selected from a list.`,
+            `Organize checkboxes in a logical order by grouping related 
+              options.`,
+          ]}
+        />
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <SubsectionText>
+          When using multiple checkboxes in a single FormField, use a
+          CheckBoxGroup. If you're using a single checkbox, use{' '}
+          <Anchor label="Checkbox" href={nameToPath('Checkbox')} />.
+        </SubsectionText>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Variants">
+        <SubsectionText>
+          CheckBox can be used individually, within a group, or as a toggle. A
```

Comment:
Since it is being called out in the copy here, should we have a variant displaying a CheckBoxGroup as toggles?

Reply:
I think I should update this copy actually. This is lingering from the individual checkbox.

Reply:
Toggle hasn't been fully implemented on CheckBoxGroup yet, so let me update the copy to reflect what is available and hold off on that example for now.

https://github.com/grommet/grommet/pull/4033